src/Confluent.Kafka/ConfigPropertyNames.cs

```diff
@@ -0,0 +1,179 @@
+namespace Confluent.Kafka
+{
+    /// <summary>
+    ///     Names of all configuration properties specific to the
+    ///     .NET Client.
+    /// </summary>
+    public static class ConfigPropertyNames
```

Comment:
Why is this class needed?
If for IDE use; that has questionable value since the librdkafka properties are not exposed that way.

Reply:
it's an efficient way to auto-document all the dotnet specific properties - they're generated along with all the other api docs.

Reply:
you made me realize the constant names are not very user-facing friendly though. fixing that up.